packages/ui-components/src/components/FormRow/FormRow.stories.js

```diff
@@ -5,7 +5,7 @@
 
 import React from "react"
 import PropTypes from "prop-types"
-import { FormRow } from "./index.js"
+import { FormRow } from "./FormRow.component"
```

Comment:
Not sure if I would like that sinse it is hard to keep track on... I do not know for me it is make sense to have more explicit import instead. First of all it would be much easy to Find and Replace staff

Reply:
Well all imports are relative so (for me, at least) it makes sense to not have to read any path and think "Okay, both components/files are in the same directory" with `./`. 🙈 
What file it's in doesn't really matter for the import of every component. And if I was looking for the component, `index` is my single source of truth is for the path. 
Otherwise I don't see much use for the export in all of the `index` files. And as we see in this migration, the more specific we make the imports, the more component imports we need to change when only a file name changes e.g. `component.js` to `component.ts` or `Row.component` to `NewRow.component `etc.
But we can talk about it in the future, just wanted to bring up as I'm seeing it often during the Typescript migration.